cmd/oci-csi-controller-driver/csioptions/csioptions.go

```diff
@@ -76,3 +90,22 @@ func NewCSIOptions() *CSIOptions {
 	}
 	return &csioptions
 }
+
+// GetFssAddress TODO : Remove this function once the value of fss-address and fss-endpoint is passed from management-plane
+// GetFssAddress returns the fssAddress based on csiAddress
+func GetFssAddress(csiAddress, defaultAddress string) string {
+	logger := zap.L().Sugar()
+	address := strings.Split(csiAddress, ".sock")
+	if len(address) != 2 || !strings.HasSuffix(csiAddress, ".sock") {
+		logger.Errorf("failed to parse csi-address : %s. Defaulting to : %s", csiAddress, defaultAddress)
+		return defaultAddress
+	}
+	fssAddress := address[0] + fssAddressSuffix
+	return fssAddress
+}
+
+// GetFssVolumeNamePrefix TODO : Remove this function once the value of fss-csi-volume-name-prefix is passed from management-plane
```

Comment:
please remove comment related to management plane

Reply:
Done